packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js

```diff
@@ -346,6 +355,10 @@ class LightCurveViewer extends Component {
 
     this.updateUserAnnotations()
   }
+  
+  isCurrentTaskValidForAnnotation () {
+    return !!this.props.currentTask && this.props.currentTask.type === 'graph2dRangeX'
```

Comment:
`!!this.props.currentTask` is presumably always true, since it defaults to a (truthy) object in the `storeMapper` function

Reply:
Wait, dangit, you're right - sorry, I was being too paranoid here, I forgot I already had a `{}` fallback when mapping `currentTask`